v21.0.0: Build fails with ngx-filesaver 21 - FileSaverModule no longer exported

## Description

`@jjmhalew/ngx-lightbox` v21.0.0 fails to build when used with `ngx-filesaver` v21.0.0 because it imports `FileSaverModule`, which was removed from `ngx-filesaver` v21.

## Error

```
✘ [ERROR] No matching export in "node_modules/ngx-filesaver/fesm2022/ngx-filesaver.mjs" for import "FileSaverModule"

    node_modules/@jjmhalew/ngx-lightbox/fesm2022/jjmhalew-ngx-lightbox.mjs:5:27
      5 │ import { FileSaverService, FileSaverModule } from 'ngx-filesaver';
```

## Root cause

`ngx-filesaver` v21.0.0 removed `FileSaverModule` and now only exports `FileSaverDirective` (standalone) and `FileSaverService`:

```js
// ngx-filesaver v21.0.0
export { FileSaverDirective, FileSaverService };
```

But `@jjmhalew/ngx-lightbox` v21.0.0 still imports `FileSaverModule` in four places:

- `fesm2022/jjmhalew-ngx-lightbox.mjs:5` — import statement
- `fesm2022/jjmhalew-ngx-lightbox.mjs:772` — `LightboxModule` NgModule imports
- `fesm2022/jjmhalew-ngx-lightbox.mjs:773` — `LightboxModule` injector imports
- `fesm2022/jjmhalew-ngx-lightbox.mjs:779` — decorator metadata

Additionally, the peer dependency in `package.json` still pins `ngx-filesaver` to the exact v20:

```json
"peerDependencies": {
  "ngx-filesaver": "20.0.0"
}
```

## Suggested fix

Replace `FileSaverModule` with `FileSaverDirective` in `LightboxModule` imports, and update the peer dependency to `^21.0.0`.

## Environment

- `@jjmhalew/ngx-lightbox`: 21.0.0
- `ngx-filesaver`: 21.0.0
- Angular: 21.x

--- a/projects/ngx-lightbox/package.json
+++ b/projects/ngx-lightbox/package.json
@@ -23,7 +23,7 @@
   "peerDependencies": {
     "@angular/common": "^22.0.0",
     "@angular/core": "^22.0.0",
-    "ngx-filesaver": "21.0.0"
+    "ngx-filesaver": "^21.0.0"
   },
   "dependencies": {
     "tslib": "^2.8.1"

